src/main/web/google27c35e6370f03e3c.html

```diff
@@ -1 +0,0 @@
-google-site-verification: google27c35e6370f03e3c.html
```

Comment:
Should we not be adding an alternative site verification code rather than just removing this one?

Reply:
No, the site is verified in other ways.  This was just to verify Ian's personal account.

Reply:
Cool cool in which case all good